### PR TITLE
List divider and Zebra Stripes

### PR DESCRIPTION
**Pull request by nilsi**
- The border that separated the list-containter and the
  archived-list-container looked a bit strange in the edge
  after the latest changes. I changed it to a separate element
  that solves it and makes it look a little bit better than the
  original one.
- Removed the gradient that was there from JqueryUI and added
  zebra stripes to the list items.

### CODE REVIEW

**Comment by nilsi** [comment]
I did not see any difference in performance.

**Comment by nilsi** [comment]
Ready for a new review

File: src/main/webapp/resources/css/styles.css [hunk 1]
```diff
@@ -126,6 +126,13 @@ p {
     background: #EDECEC;
 }
 
+#list-divider {
+    border-bottom: 9px solid #494949;
+    width: -moz-calc(100% + 8px);
+    width: -webkit-calc(100% + 8px);
+    width: calc(100% + 8px);
```
Comment (fredriklindman) [inline src/main/webapp/resources/css/styles.css:133]: Nice with "calc"!

File: src/main/webapp/resources/js/scripts.js [hunk 2]
```diff
@@ -2655,6 +2659,20 @@ $(document).ready(function () {
...
     });
+    
+    /**
+     * This method alternate the color of parent li's.
+     * For a readability purpose
+     */
+    var addZebraStripesToParents = function() { 
+    	$( "#list-container .parentLi" ).each(function(index) {
+	       if(index % 2 == 0) {
+	    	   $(this).css("background","#f9f9f9");
```
Comment (fredriklindman) [inline src/main/webapp/resources/js/scripts.js]: Could this be changed into adding a CSS-class for the item with that background? Currently it creates the following inline style which looks a bit too long ;)
style="background: none repeat scroll 0% 0% rgb(249, 249, 249);

**Comment by fredriklindman** [comment]
1. I think the zebra color is a little too similar to the archived-color. Have you tried experimenting with other colors than gray? I'm not sure but it somehow feels a little color-less. Maybe it should be like that but I think there is room for experimenting. Do you personally prefer it to be gray? Maybe the white li's could get a little tone as well (not much though). Perhaps this is something we should discuss with more people to see what they think.
2. When archiving stories with several in edit mode, a redraw of stripes should be triggered.

File: src/main/webapp/resources/js/scripts.js (hunk 2 above)
Comment (nilsi) [inline src/main/webapp/resources/js/scripts.js]: Good point, it's changed!

**Comment by nilsi** [comment]
1. Yes I have been experimenting with it and I did not like the other colors I tested. Its really easy to experiment with directly from chrome, if you have any suggestion on other colors, feel free to test it out. However I changed the tone of the archived li's.
2. Done.

**Comment by fredriklindman** [comment]
I was just about to merge when I thought about the children. Shouldn't they either be zebra-striped as well or have the same color as their parent? I think I vote for the latter.

**Comment by fredriklindman** [comment]
One more thing, sorry I didn't find it before:
When I press expand/collapse all, the stripes disappear.

EDIT: And the same when using the filter

**Pull request merged**